server/boards/routes.ts

```diff
@@ -130,6 +131,60 @@ router.get("/:board_id", ensureBoardAccess, async (req, res) => {
     );
 });
 
+/**
+ * @openapi
+ * /boards:
+ *   post:
+ *     summary: Create board.
+ *     operationId: createBoard
+ *     tags:
+ *       - /boards/
+ *     requestBody:
+ *       description: request body
```

Comment:
Put in the actual description of the body of the request. This would be something like "the details of the new board to be created"